interpreter/shared/src/main/scala/sigmastate/eval/CostingDataContext.scala

```diff
@@ -25,12 +25,13 @@ import scalan.{Nullable, RType}
 import scorex.crypto.hash.{Blake2b256, Digest32, Sha256}
 import sigmastate.Values.ErgoTree.EmptyConstants
 import sigmastate.basics.DLogProtocol.ProveDlog
-import sigmastate.basics.{ProveDHTuple, CryptoConstants}
+import sigmastate.basics.{CryptoConstants, ProveDHTuple}
 import sigmastate.crypto.{CryptoFacade, Ecp}
 import sigmastate.lang.TransformingSigmaBuilder
```

Comment:
import sigmastate.lang.TransformingSigmaBuilder and 
import scalan.Nullable

are not used now

Reply:
fixed